spinn_machine/chip.py

```diff
@@ -60,7 +60,9 @@ def __init__(self, x, y, processors, router, sdram, nearest_ethernet_x,
         :type ip_address: str
         :param virtual: boolean which defines if this chip is a virtual one
         :type virtual: bool
-        :param tag_ids: ID to identify the chip for SDP
+        :param tag_ids: IDs to identify the chip for SDP can be empty to
+            define no tags or None to allocate tag automatically
+            based on if there is an ip_address
         :type tag_ids: iterable(int)
```

Comment:
or none, otherwise pycharm will complain with incorrect python docs

Reply:
ok fiddle it